### PR TITLE
Fix #3456: Merge all adminstrator_controls_* file into individual xml files.

### LINKED ISSUE #3456: Merge all `adminstrator_controls_*` file into individual xml files.
Currently there are 3 versions of each of these files:
- `administrator_controls_account_action_view`
- `administrator_controls_activity`
- `administrator_controls_app_information_view`
- `administrator_controls_fragment`
- `administrator_controls_general_view`
- `administrator_controls_profile_view`

We need to merge this different versions into individual single files. The PR should reduce 3*6 = `18 files` to `6 files`.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by aayushimathur6**
## Explanation
Fixes #3456 

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

| Before  | After |
| ------- | ------- |
| Screenshot_20220228_155829 | Screenshot_20220228_180530 |
| Screenshot_20220228_155903 | Screenshot_20220228_180549 |
| Screenshot_20220228_155938 | Screenshot_20220228_180617 |
| Screenshot_20220228_170135 | Screenshot_20220228_180636 | 
| Screenshot_20220228_175354 | Screenshot_20220228_180128 |
| Screenshot_20220228_175656  | Screenshot_20220228_180158 |
| Screenshot_20220228_175740 | Screenshot_20220228_180043 |

### CODE REVIEW

**Comment by aayushimathur6** [comment]
hey @ayush0402 please review it!!

File: app/src/main/res/values-sw600dp-port/dimens.xml [hunk 1]
```diff
@@ -438,4 +438,21 @@
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_start">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_end">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_widget_switch_compat_margin_end">4dp</dimen>
+
+  <!-- AdministratorControlAccountActionsView  -->
+  <dimen name="administrator_control_account_actions_view_account_actions_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_control_account_actions_view_account_actions_text_view_padding_end">16dp</dimen>
+  <dimen name="administrator_control_account_actions_view_log_out_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_control_account_actions_view_log_out_text_view_padding_end">16dp</dimen>
```
Comment (ayush0402) [inline app/src/main/res/values-sw600dp-port/dimens.xml]: I guess we don't need to introduce duplicate entries from values/dimens.xml to other files. Because, in case these values are not here, it would automatically reference to the same variable under values/dimens.xml. Ditto for other places.

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -648,4 +648,32 @@
...
+  <dimen name="administrator_control_account_actions_view_log_out_text_view_padding_end">16dp</dimen>
+
+  <!-- AdministratorControlsAppInformationView -->
+  <dimen name="administrator_controls_app_information_view_app_information_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_controls_app_information_view_app_information_text_view_padding_end">16dp</dimen>
+  <dimen name="administrator_controls_app_information_view_app_version_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_controls_app_information_view_app_version_text_view_padding_end">16dp</dimen>
+
+  <!-- AdministratorControlsFragment -->
+  <dimen name="administrator_controls_fragment_padding_bottom">80dp</dimen>
```
Comment (ayush0402) [inline app/src/main/res/values/dimens.xml]: I think it would be better renamed to `administrator_controls_fragment_recycler_view_padding_bottom`.

File: app/src/main/res/layout/administrator_controls_general_view.xml [hunk 3]
```diff
@@ -18,9 +18,9 @@
       style="@style/Heading5ViewStart"
       android:layout_width="match_parent"
       android:background="@color/admin_controls_sub_heading_background_color"
-      android:paddingStart="16dp"
-      android:paddingTop="20dp"
-      android:paddingEnd="16dp"
+      android:paddingStart="@dimen/administrator_controls_general_view_general_text_view_padding_start"
+      android:paddingTop="@dimen/administrator_controls_general_view_general_text_view_padding_top"
+      android:paddingEnd="@dimen/administrator_controls_general_view_general_text_view_padding_end"
```
Comment (ayush0402) [inline app/src/main/res/layout/administrator_controls_general_view.xml:23]: I am not sure if this way of naming dimens is acceptable or not. I think something like `administrator_controls_general_view_sub_heading_text_view_padding_start` and `administrator_controls_general_view_options_text_view_padding_start` (for lower text view) will be better for consistency with the colors.

**Review by ayush0402 — CHANGES_REQUESTED** [review]
@aayushimathur6 Overall this LGTM, just had a few suggestions.

File: app/src/main/res/values/dimens.xml [hunk 4]
```diff
@@ -648,4 +648,32 @@
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_start">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_end">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_widget_switch_compat_margin_end">4dp</dimen>
+
+  <!-- AdministratorControlAccountActionsView  -->
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml]: Nit fix. Remove extra space after View
`<!-- AdministratorControlAccountActionsView -->`

File: app/src/main/res/values-sw600dp-port/dimens.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/values-sw600dp-port/dimens.xml]: Agreed. All values for `administrator_control_account_actions_view_*` can be removed from this file and `sw600dp-land/dimens.xml` file too.

File: app/src/main/res/layout/administrator_controls_general_view.xml (hunk 3 above)
Comment (rt4914) [reply app/src/main/res/layout/administrator_controls_general_view.xml:23]: @ayush0402 Don't think of it in terms of color because that is something we cannot guarantee at every point and therefore I think the current naming is correct.

File: app/src/main/res/values-sw600dp-land/dimens.xml [hunk 5]
```diff
@@ -439,4 +439,21 @@
...
+
+  <!-- AdministratorControlsFragment -->
+  <dimen name="administrator_controls_fragment_padding_bottom">80dp</dimen>
+
+  <!-- AdministratorControlsGeneralView -->
+  <dimen name="administrator_controls_general_view_general_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_controls_general_view_general_text_view_padding_top">20dp</dimen>
+  <dimen name="administrator_controls_general_view_general_text_view_padding_end">16dp</dimen>
+  <dimen name="administrator_controls_general_view_edit_account_text_view_padding_start">16dp</dimen>
+  <dimen name="administrator_controls_general_view_edit_account_text_view_padding_end">16dp</dimen>
```
Comment (rt4914) [inline app/src/main/res/values-sw600dp-land/dimens.xml]: These dimens can be removed for `AdministratorControlsGeneralView` from this file as they match with that of `values/dimens.xml` file.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@aayushimathur6 Suggested changes. Thanks for a very clean PR.

**Comment by aayushimathur6** [comment]
@ayush0402 please review it

File: app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (aayushimathur6) [reply app/src/main/res/values/dimens.xml]: done

File: app/src/main/res/values/dimens.xml (hunk 4 above)
Comment (aayushimathur6) [reply app/src/main/res/values/dimens.xml]: done

File: app/src/main/res/values-sw600dp-land/dimens.xml (hunk 5 above)
Comment (aayushimathur6) [reply app/src/main/res/values-sw600dp-land/dimens.xml]: done

**Comment by aayushimathur6** [comment]
Updated my PR @rt4914 , @ayush0402

File: app/src/main/res/values-land/dimens.xml [hunk 6]
```diff
@@ -448,7 +448,36 @@
...
+  <!-- AdministratorControlsProfileView -->
+  <dimen name="administrator_controls_profile_view_profile_management_text_view_padding_start">36dp</dimen>
+  <dimen name="administrator_controls_profile_view_profile_management_text_view_padding_end">36dp</dimen>
+  <dimen name="administrator_controls_profile_view_edit_profiles_text_view_padding_start">36dp</dimen>
+  <dimen name="administrator_controls_profile_view_edit_profiles_text_view_padding_end">36dp</dimen>
+
   <!-- OptionAudioLanguage -->
   <dimen name="option_audio_language_padding_start">36dp</dimen>
   <dimen name="option_audio_language_padding_end">36dp</dimen>
+
```
Comment (ayush0402) [inline app/src/main/res/values-land/dimens.xml]: Remove this extra line.

**Review by ayush0402 — APPROVED** [review]
LGTM, just a nit change.

File: app/src/main/res/values-sw600dp-port/dimens.xml [hunk 7]
```diff
@@ -438,8 +438,4 @@
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_start">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_end">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_widget_switch_compat_margin_end">4dp</dimen>
-
-  <!-- OptionAudioLanguage -->
-  <dimen name="option_audio_language_padding_start">16dp</dimen>
-  <dimen name="option_audio_language_padding_end">16dp</dimen>
```
Comment (rt4914) [inline app/src/main/res/values-sw600dp-port/dimens.xml:447]: Why are these getting removed? They don't seem to be part of this PR? Maybe merging with latest develop can fix this automatically.

File: app/src/main/res/values-sw600dp-land/dimens.xml [hunk 8]
```diff
@@ -439,8 +439,4 @@
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_start">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_constraint_layout_padding_end">16dp</dimen>
   <dimen name="administrator_controls_download_permissions_view_auto_update_topic_widget_switch_compat_margin_end">4dp</dimen>
-
-  <!-- OptionAudioLanguage -->
-  <dimen name="option_audio_language_padding_start">16dp</dimen>
-  <dimen name="option_audio_language_padding_end">16dp</dimen>
```
Comment (rt4914) [inline app/src/main/res/values-sw600dp-land/dimens.xml:448]: Why are these getting removed? They don't seem to be part of this PR? Maybe merging with latest develop can fix this automatically.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@aayushimathur6 Minor changes. Merge with latest develop.

File: app/src/main/res/values-land/dimens.xml (hunk 6 above)
Comment (aayushimathur6) [reply app/src/main/res/values-land/dimens.xml]: done

File: app/src/main/res/values-sw600dp-port/dimens.xml (hunk 7 above)
Comment (aayushimathur6) [reply app/src/main/res/values-sw600dp-port/dimens.xml:447]: done

File: app/src/main/res/values-sw600dp-land/dimens.xml (hunk 8 above)
Comment (aayushimathur6) [reply app/src/main/res/values-sw600dp-land/dimens.xml:448]: done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.
